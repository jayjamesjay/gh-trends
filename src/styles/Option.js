import styled from 'styled-components';

const Option = styled.option`
  font-size: 1.2rem;
  text-transform: uppercase;
  line-break: loose;
  text-align: center;
`;

export default Option;
