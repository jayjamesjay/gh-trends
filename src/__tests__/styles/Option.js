import Option from '../../styles/Option';
import 'jest-styled-components';

it('renders default <Option/>', () => {
  const wrapper = shallow(<Option />);
  expect(wrapper).toMatchSnapshot();
});
