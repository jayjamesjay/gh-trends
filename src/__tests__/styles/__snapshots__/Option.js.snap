// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default <Option/> 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "Option-sc-1is677r-0",
        "isStatic": true,
        "rules": Array [
          "font-size:1.2rem;line-break:loose;text-align:center;text-transform:uppercase;",
        ],
      },
      "displayName": "Option",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "Option-sc-1is677r-0",
      "target": "option",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
